addons/account/demo/account_demo.py

```diff
@@ -95,6 +96,32 @@ def _post_load_demo_data(self, company=False):
             except (UserError, ValidationError):
                 _logger.exception('Error while posting demo data')
 
+        # We want the initial balance to be set in he equity_unaffected account
+        demo_bank_statement_1 = self.ref('demo_bank_statement_1')
+        if demo_bank_statement_1:
```

Comment:
```suggestion
        demo_bank_statement_1 = self.ref('demo_bank_statement_1', raise_if_not_found=False)
        if demo_bank_statement_1:
```
Otherwise the `ref()` will raise if `demo_bank_statement_1` does not exist (same below).